### PR TITLE
 feat(forms) : Import form basic UI

### PR DESCRIPTION
**Pull request by AdrienClairembault**
Minimal import UI.

Step 1:

image

Step 2:

image

Step 3:

image

Note that the second step isn't very interesting right now as it display something very similar to the third step.
This is temporary, in the future it will allow the user to pick replacements for missing items.

| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets |

### CODE REVIEW

**Comment by AdrienClairembault** [comment]
Need #17644 to be merged first.

**Comment by AdrienClairembault** [comment]
Rebased to latest main (PHPStan level 4 was checked)

File: templates/pages/admin/form/import/step3_execute.html.twig [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+            <thead>
+                <tr>
+                    <th class="w-70 px-4">{{ "Glpi\\Form\\Form"|itemtype_name }}</th>
+                    <th class="w-30 px-4">{{ __("Status") }}</th>
+                </tr>
+            </thead>
+            <tbody>
+                {% for form in results.getImportedForms() %}
+                    <tr>
+                        <td class="w-70 px-4">{{ form.getLink()|raw }}</td>
```
Comment (cedric-anne) [inline templates/pages/admin/form/import/step3_execute.html.twig:57]: It should work and prevent the usage of the `|raw` filter.

```suggestion
                        <td class="w-70 px-4">{{ get_item_link(form) }}</td>
```

**Review by cedric-anne — COMMENTED** [review]
Please rebase to fix conflicts.

**Review by cedric-anne — APPROVED** [review]

**Pull request merged**